src/plugins/intel_cpu/src/plugin.h

```diff
@@ -61,9 +61,12 @@ class Engine : public InferenceEngine::IInferencePlugin {
 
     Config engConfig;
     ExtensionManager::Ptr extensionManager = std::make_shared<ExtensionManager>();
-    /* Explicily configured streams have higher priority even than performance hints.
+    /* Explicily configured streams have higher priority than performance hints.
        So track if streams is set explicitly (not auto-configured) */
     bool streamsExplicitlySetForEngine = false;
+    /* Explicily configured inference_precision has higher priority than performance hints.
+       So track if inference_precision is set explicitly (not auto-configured) */
+    bool inferencePrecisionSetForEngine = false;
```

Comment:
should not this information be part of `Config`?

Reply:
Yes, I have left a todo comment at the piece of code when this parameter is set, that we should store both value and "if_set_explicitly" flag in our Config.
The idea is to do major refactoring of the config + plugin files after we drop legacy configuration.

Reply:
Changed my mind :)
we can actually extend manualEnforceBF16 a bit to be also usable in the context of execution_mode.
This eliminates some extra code.